recipe/meta.yaml

```diff
@@ -28,7 +28,7 @@ requirements:
     build:
         - python
         - setuptools
-        - toolchain
+        - mkl
```

Comment:
@msarahan is there a way to ensure that `numpy` with `mkl` is installed? This does not seem to work.